docs/developers/packaging.md

```diff
@@ -256,10 +260,10 @@ it to the final napari installer:
 
 Very fun! So where do all these packages live?
 
-| Package            | Fork                                            | Feedstock                                          |
+| Package            | Source                                          | Feedstock                                          |
 |--------------------|-------------------------------------------------|----------------------------------------------------|
-| `constructor`      | [jaimergp/constructor @ `menuinst+branding`][9] | [jaimergp-forge/constructor-feedstock][12]         |
-| `conda-standalone` | _N/A_                                           | [conda-forge/conda-standalone-feedstock PR#21][13] |
+| `constructor`      | [jaimergp/constructor @ `menuinst-cep`][9]      | [jaimergp-forge/constructor-feedstock][12]         |
+| `conda-standalone` | _Same as feedstock_.                            | [conda-forge/conda-standalone-feedstock PR#21][13] |
```

Comment:
Can't comment there, but a few lines below there is:
```
Most of the forks live in `jaimergp`'s account, under a non-default branch. They are published
through the `jaimergp-forge` every time a commit to `master` is made. Versions are arbitrary here,
```
Should that be `main` and not `master`?

Reply:
Some of those forks haven't been migrated to use `main` yet 😬

Reply:
But yes, we can say `main` and it wouldn't hurt the docs.